src/main/java/io/stargate/sgv2/jsonapi/config/DocumentLimitsConfig.java

```diff
@@ -13,15 +13,26 @@
 @ConfigMapping(prefix = "stargate.jsonapi.document.limits")
 public interface DocumentLimitsConfig {
 
-  /** Defines the max size of filter fields, default is 64 fields. */
+  /** Defines the default max size of filter fields. */
```

Comment:
Adding constants (refactoring) because we cannot inject `DocumentLimitsConfig` into integration tests.